### DISCUSSION TITLE
Initial Discussions

### DISCUSSION THREAD

**Discussion by rtshkmr** [ve1ld/vyasa#2] [Ideas]
## Follow up actions: 

1. Early Adopters: 
	 - the gap that a text-based interface creates is not a priority right now 
	 - we shall coattail on other platforms: e.g. for youtube videos, ride onto comments that you see there. E.g. there are T-series videos with huge viewership numbers ==> can get commentary + info with nice overlay 
	 
	   
2. Some key engines: 
	 - archival engine 
	 - datastructs: 
	    - Representing verses and creating a corpus: 
	       * schematising the data verse by verse 
	          * multiple views of the central data obj:
	             - translation view 
	             - 
3. Requirements to Strive for: 
   * sleek interface is a req esp since it's a text based interface: https://write.sonnet.io/ 
   
   
   

## References Related to the Initial Inspiration: 
1. Examples of Digital Archives from various cultures: 
    * [Ctext Project](https://ctext.org/) is a digital archive of pre-mordern chinese literature.
    * [Tamil Virtual Academy ](https://www.tamilvu.org/ta/library-l3700-html-l3700ind-133880)
    * There's an [IIT project](https://www.gitasupersite.iitk.ac.in/) about the gita, but it doesn't look that modern
2. Sleek text-based UI inspirations: 
    * [Annotations plugin](https://web.hypothes.is/help/annotation-basics/) with sharing, public discussion thread features 
    * Gwern Branwen's [Website](https://gwern.net/me#software): it's actually pretty appealing. The hyperlinks seem to be autogenerated though, e.g. simple stuff like "Cat" is wiki-linked.
      * Colours [Rubrication](https://gwern.net/red) as a means of standing out
3. Architectural Patterns & Data Structures of note:
    * [Pattern Language -- Project Xanadu](https://maggieappleton.com/xanadu-patterns)
      * 1. Visible links (similar to wikipedia) 
4. Some noteworthy trends in the internet: 
    * Digital Gardening: 
        - A dive into the practice of growing digital gardens ([here](https://maggieappleton.com/garden-history))
        - [examples](https://twitter.com/Mappletons/status/1250532315459194880) of digital garden
     ** https://www.are.na/

**Comment by rtshkmr** [comment]
### Uncat Implementation Points: 

#### 1. Youtube &  iframe embeds: 
- [iframe player api ref](https://developers.google.com/youtube/iframe_api_reference)

@KosmonautX mentioned: 


so if we are building ontop of youtube embeds

we can manipulate the video frame

and pair it with our parallel corpus

https://youtu.be/e4vxg2stIIA

will essentially digitise the problem channels like his face having to create multiple frames of slides to transpose this material

**Comment by rtshkmr** [comment]
# UI Inspirations

## Some Areas of Focus for UI: 
1. text based UI (e.g. magazines...)
2. Integration with existing journeys e.g. whatsapp forwarded messages 

#### Dump: 
1. Design guidelines for dealing with information density -- https://blog.logrocket.com/balancing-information-density-in-web-development/ 
   - I think the information density should be properly managed else it will be overwhelming for a user
<details><summary>2. Spotify Desktop's 3 panel split is something that could be followed since it contains text-based info along with media controls and interaction </summary>
<img width="1680" alt="image" src="https://github.com/ve1ld/vyasa/assets/38996397/dd018d07-5706-4ef8-b048-003c5d141d08">



</details> 
  3. Hello Monday has really good use of whitespaces -- https://www.hellomonday.com/ .

**Comment by rtshkmr** [comment]
# Entry Gateways

## Consumption of Video 

1. serving the general person, by value-adding their consumption of common things (videos) ==> this is a good entry-gateway 
	* the example here is the [youglish](https://youglish.com/)
2. Using killer apps like whatsapp, to integrate with features like stories as well

**Reply by ks0m1c** [reply]
Url fragments and breadcrumbs

**Comment by rtshkmr** [comment]
# Case Study: TheQuran.com

https://quran.com/25?startingVerse=6




![image](https://github.com/ve1ld/vyasa/assets/38996397/c8b56ba0-f54a-4420-abad-2c7d16db17c7)

**Comment by rtshkmr** [comment]
# Case Study: SuttaCentral and Friends (Buddhanexus)

- https://suttacentral.net/?lang=en 
- https://buddhanexus.net/ ==> they figured out how to cross link across texts (and across languages of similar content)
